tests/Elastic.CommonSchema.NLog.Tests/LogTestsBase.cs

```diff
@@ -22,23 +23,27 @@ private class EventInfoMemoryTarget : TargetWithContext
 			protected override void Write(LogEventInfo logEvent) => Events.Add(logEvent);
 		}
 
-		protected List<string> ToFormattedStrings(IEnumerable<LogEventInfo> logEvents) =>
+		protected List<string> ToFormattedStrings(List<LogEventInfo> logEvents) =>
 			logEvents
 				.Select(l => new EcsLayout().Render(l))
 				.ToList();
 
-		protected List<(string Json, Base Base)> ToEcsEvents(IEnumerable<LogEventInfo> logEvents) =>
+		protected List<(string Json, Base Base)> ToEcsEvents(List<LogEventInfo> logEvents) =>
 			ToFormattedStrings(logEvents)
 				.Select(s => (s, Base.Deserialize(s)))
 				.ToList();
 
 		protected static void TestLogger(Action<ILogger, Func<List<LogEventInfo>>> act)
 		{
 			var configurationItemFactory = new Config.ConfigurationItemFactory();
-			configurationItemFactory.LayoutRenderers.RegisterDefinition("ElasticApmTraceId", typeof(ApmTraceIdLayoutRenderer));
-			configurationItemFactory.LayoutRenderers.RegisterDefinition("ElasticApmTransactionId", typeof(ApmTransactionIdLayoutRenderer));
+			configurationItemFactory.LayoutRenderers.RegisterDefinition(ApmTraceIdLayoutRenderer.Name, typeof(ApmTraceIdLayoutRenderer));
+			configurationItemFactory.LayoutRenderers.RegisterDefinition(ApmTransactionIdLayoutRenderer.Name, typeof(ApmTransactionIdLayoutRenderer));
 			configurationItemFactory.RegisterItemsFromAssembly(Assembly.GetAssembly(typeof(EcsLayout)));
 
+			// These layout renderers need to registered statically
```

Comment:
might be good to explain a bit further why they need to be registered on the static `LayoutRenderer`

Reply:
Sure, thats fair.